apps/github/src/@types/index.d.ts

```diff
@@ -0,0 +1,30 @@
+type Payload = {
+    action: string;
+    sender: { [key: string]: unknown };
+    repository: { [key: string]: unknown };
+    organization: { [key: string]: unknown };
+    installation: { [key: string]: unknown };
+    [key: string]: unknown;
```

Comment:
So, we don't know the data type of these properties? As the value is set to unknown.

Reply:
This type can be constructed using Record utility type.

so, `[key: string]: unknown` => `Record<string, unknown>`

Reply:
> 

![image](https://user-images.githubusercontent.com/44132059/213742093-5501a50a-e4e1-40c8-a9b1-d40f25854fbd.png)
Github has labelled the type as object so I used unknown.

Reply:
> This type can be constructed using Record utility type.
> 
> so, `[key: string]: unknown` => `Record<string, unknown>`

Done.